<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderListWidget</class>
 <widget class="QWidget" name="OrderListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbn_backMenu">
        <property name="text">
         <string>退菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbn_hurryMenu">
        <property name="text">
         <string>催菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnOk">
        <property name="text">
         <string>提交</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbn_accounts">
        <property name="text">
         <string>结账</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
